src/Components/Modal/Readme.md

```diff
@@ -143,6 +143,60 @@ function ModalExample() {
 <ModalExample />;
 ```
 
+### Large
+
+When you need to show a modal with a custom header, provide a custom header. Title will be ignored.
+
+```js
+import { useState } from 'react';
+import Button from '../Button/Button';
+
+function ModalExample() {
+  const [showModal, setShowModal] = useState(false);
+
+  const LargeModal = (
+    <Modal
+      large
+      mobileFullScreen
+      header={
+        <Block direction="row">
```

Comment:
missing Block and Heading from import causing error 
![image](https://user-images.githubusercontent.com/41709608/140436739-c45a3efd-1aa4-4ff8-b3fc-e96db74edb1b.png)